src/core/services/theming/theming.js

```diff
@@ -1003,6 +1000,48 @@ function generateAllThemes($injector, $mdTheming) {
       }
     });
   }
+
+  // Split a string representing a CSS file producing an array with a rule in each element of it.
+  function splitCss(themeCss) {
+    var result = [];
+    var currentRule = '';
+    var openedCurlies = 0;
+    var closedCurlies = 0;
+
+    for (var i = 0; i < themeCss.length; i++) {
+      var character = themeCss.charAt(i);
+
+      // Check for content in quotes
+      if (character === '\'' || character === '\"') {
+        // Find closing quote
+        var nextQuoteIndex = themeCss.indexOf(character, i + 1);
+        var textInQuotes = themeCss.substring(i, nextQuoteIndex);
+        currentRule += textInQuotes;
```

Comment:
We can merge this line with the declaration above. Instead, just add a comment that we're appending the quoted text to the current rule.